packages/transformer/src/test/standalone/IModelTransformerHub.test.ts

```diff
@@ -1040,5 +1041,113 @@ describe("IModelTransformerHub", () => {
     await tearDown();
     sinon.restore();
   });
+
+  // unskip this and set the env vars "REMOTE_IMODEL_DIR" to a path and "FORK_CHANGESET_ID" to a
+  // changeset id, to locally test synchronizing an remote iModel to a fork
+  // You must have used some kind of imodel-downloader tool to have all changesets stored locally
+  it.only("should test transforming a remote iModel to local", async () => {
```

Comment:
Need to change this to .skip, just keeping it as is for someone else who is using this branch